res/values-hi/aosip_strings.xml

```diff
@@ -0,0 +1,201 @@
+
+    <!-- Fingerprint location message - generic -->
+    <string name="security_settings_fingerprint_enroll_find_sensor_message_front" product="device">फिंगरप्रिंट सेंसर का अपने डिवाइस की स्क्रीन पर स्थान निर्धारण करें।</string>
+    <string name="security_settings_fingerprint_enroll_find_sensor_message_left" product="device">फिंगरप्रिंट सेंसर का अपने डिवाइस के बाईं ओर स्थान निर्धारण करें।</string>
+    <string name="security_settings_fingerprint_enroll_find_sensor_message_right" product="device">फिंगरप्रिंट सेंसर का अपने फोन के दाईं ओर स्थान निर्धारण करें।</string>
+
+    <!-- Fingerprint location message - tablet -->
+    <string name="security_settings_fingerprint_enroll_find_sensor_message_front" product="tablet">अपने टैबलेट के फ्रंट पर फिंगरप्रिंट सेंसर का स्थान निर्धारण करें।</string>
+    <string name="security_settings_fingerprint_enroll_find_sensor_message_left" product="tablet">अपने टैबलेट के बाईं ओर फिंगरप्रिंट सेंसर का स्थान निर्धारण करें।</string>
+    <string name="security_settings_fingerprint_enroll_find_sensor_message_right" product="tablet">अपने टेबलेट के दाईं ओर फ़िंगरप्रिंट सेंसर का स्थान निर्धारण करें।</string>
+
+    <!-- DerpQuest Settings strings -->
+    <string name="derp_settings_title">डर्पक्वेस्ट</string>
+    <string name="derp_settings_summary">क्रोनिक में हमें भरोसा है ...</string>
+    <string name="derp_extras_qs_title">त्वरित सेटिंग्स</string>
+
+    <!-- Quick unlock -->
+    <string name="unlock_quick_unlock_control_title">त्वरित अनलॉक</string>
+    <string name="unlock_quick_unlock_control_summary">सही पिन/पासवर्ड दर्ज करने पर स्वचालित रूप से अनलॉक करें</string>
+
+    <!-- Custom light color -->
+    <string name="custom_light_title">हल्का रंग</string>
+    <string name="custom_light_on_time_title">एलईडी पल्स के दौरान प्रकाश समय (मिली सेकंड)</string>
+    <string name="custom_light_off_time_title">एलईडी बंद समय के दौरान पल्स (मिली सेकंड)</string>
+
+    <!-- String for removal of sensitive info on about, depending on tap -->
+    <string name="device_info_protected_single_press">जानकारी दिखाने के लिए टैप करें</string>
+
+    <!-- Sound: Title for the option defining the phone ringtone for slot1 on Muilti SIM device. [CHAR LIMIT=50] -->
+    <string name="ringtone1_title">फोन रिंगटोन - सिम 1</string>
+
+    <!-- Sound: Title for the option defining the phone ringtone for slot2 on Muilti SIM device. [CHAR LIMIT=50] -->
+    <string name="ringtone2_title">फोन रिंगटोन - सिम 2</string>
+
+    <!-- Charging vibration -->
+    <string name="charging_vibro_title">चार्जिंग कंपन</string>
+
+    <!-- Notification sound timeout -->
+    <string name="app_notification_sound_timeout_title">सूचना ध्वनियों के बीच न्यूनतम समय</string>
+    <string name="app_notification_sound_timeout_summary_template">ध्वनियों या कंपन को एक से अधिक बार हर <xliff:g id="duration">%1$s</xl:> में अनुमति दें</string>
```

Comment:
```suggestion
    <string name="app_notification_sound_timeout_summary_template">ध्वनियों या कंपन को एक से अधिक बार हर <xliff:g id="duration">%1$s</xliff:g></string>
```
This was not done correctly

Reply:
Yupp, messed up there but the last three words should be there.

Reply:
edited. like this?

Reply:
Yes, thats better now